recipes/pbtk/meta.yaml

```diff
@@ -26,6 +26,7 @@ extra:
 build:
   number: 0
   skip: True  # [osx]
+  binary_relocation: False
```

Comment:
Can you explain why this is needed for your packages?

Reply:
We have had to add this previously to our packages to resolve build issues:
https://github.com/bioconda/bioconda-recipes/pull/44388